src/api/app/views/webui2/webui/monitor/_workers_table.html.haml

```diff
@@ -21,20 +21,19 @@
   %p
     %i No workers
 - else
-  - workers_sorted.each_slice(5) do |row|
-    %ul.monitorboxrow
-      %li.hidden
-        - row.each do |name, hash|
-          %li.builderbox
-            .builderboxtitle #{name} (#{hash['_arch']})
-            - hash.each do |subid, id|
-              - if subid != '_arch'
-                .monitorpb{ id: "p#{id}" }
-                  .progress
-                    .progress-bar.progress-bar-striped{ 'aria-valuemax' => '100', 'aria-valuemin' => '0',
-                                                        'role' => 'progressbar' }
+  .d-flex.flex-wrap
```

Comment:
We could get rid of all the classes and target the tags directly by applying an id to the div on this line . The class names are weird and the CSS in `monitor.scss` could be more concise. The margin, padding, list-style, font-weight and probably more could be Bootstrap classes.

Let me know what you think. I can help with this :slightly_smiling_face:

Reply:
We agreed on doing this in another time.